physical/raft/raft.go

```diff
@@ -1391,6 +1402,9 @@ func (b *RaftBackend) SetupCluster(ctx context.Context, opts SetupOpts) error {
 		}
 	}
 
+	if opts.RemovedCallback != nil {
+		b.removedCallback = opts.RemovedCallback
```

Comment:
Should this be a call to `SetRemovedCallback` instead? (Genuine question, I am not sure if we care about locking at this point.)

Reply:
the lock is already grabbed earlier in the function, so we shouldn't grab it again

Reply:
Yep, missed that 🤦